src/_posts/languages/nodejs/2000-01-01-start.md

```diff
@@ -356,6 +356,118 @@ To fix this issue, modify the `start` script by adding `-p $PORT`
 }
 ```
 
+### Next.js in Standalone Mode
```

Comment:
suggestion: I don't think this is a good place for this section. The page is already quiet large. I would add it to this page (https://doc.scalingo.com/platform/app/slugignore) as it already contains information about reducing image size. And on this page only add a word with a link to the slugignore page. What do you think?

Reply:
I've added a dedicated page https://doc.scalingo.com/languages/nodejs/nextjs-standalone and linked it from:
* https://doc.scalingo.com/platform/app/slugignore#nextjs-application
* https://doc.scalingo.com/languages/nodejs/start#standalone-mode